src/python/ensembl/io/genomio/seq_region/report.py

```diff
@@ -0,0 +1,70 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Report record."""
+
+__all__ = [
+    "ReportRecord",
+]
+
+import csv
+from os import PathLike
+from pathlib import Path
+import re
+from typing import Any, Tuple
+
+from ensembl.utils.archive import open_gz_file
+
+##############################################
+SeqRegionDict = dict[str, Any]
```

Comment:
```suggestion
SeqRegionDict = TypeVar(dict[str, Any])
```
To create types is better to use `TypeVar` (and do not forget `from typing import TypeVar`!).